content/patterns/button/examples/button.html

```diff
@@ -195,6 +195,18 @@ <h2 id="sc1_label">HTML Source Code</h2>
           sourceCode.make();
         </script>
       </section>
+
+
+      <section>
+        <h2>Support Levels</h2>
+        <iframe
+          class="support-levels-command-button"
+          src="https://aria-at-app-sandbox.bocoup.com/embed/reports/command-button"
```

Comment:
```suggestion
          src="https://aria-at.w3.org/embed/reports/command-button"
```
As https://github.com/w3c/aria-at-app/pull/476 has been merged and available in production.

This should also address @mcking65 comment that:

> https://deploy-preview-180--aria-practices.netlify.app/aria/apg/patterns/button/examples/button/#supportlevels gives me
> >502 Bad Gateway

As the sandbox environment was experiencing performance issues.